### PR TITLE
Add Heliostation (Working title)

### PR DESCRIPTION
**Pull request by remove32**
## About The Pull Request

Add Heliostation, a map designed by Helianthus.

This map is larger than box and should be able to better support our average population.
The hallways are spacious and it should be easy to navigate to each area.
heliostationmap

Feedback thread: https://fulpstation.com/showthread.php?tid=1470

## Why It's Good For The Game
Primarily we want to have our own map - this is the first step towards accomplishing that. 

## Changelog
:cl:
add: New map - Heliostation!
/:cl:

### CODE REVIEW

File: _maps/heliostation.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "map_name": "Heliostation",
+    "map_path": "map_files/Heliostation",
+    "map_file": "heliostation.dmm",
+    "shuttles": {
+	    "cargo": "cargo_box",
+	    "ferry": "ferry_fancy",
+	    "whiteship": "whiteship_box",
+	    "emergency": "emergency_box"
```
Comment (EgorDinamit) [inline _maps/heliostation.json:9]:
```suggestion
	    "emergency": "emergency_news"
```

Comment (Homingpenguins) [reply _maps/heliostation.json:9]: is that all i had to do to add that...

**Comment by Samuuu3** [comment]
I really love this map.

**Comment by Fikou** [comment]
not a fan of the giant 5 wide hallways with no decoration except colored tiles on the far edges, but other than that the map seems fine

**Pull request merged**